docs/it/ciclo-del-feedback.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Ciclo del Feedback
+nav_order: 999
+---
+
+# Il Ciclo del Feedback
+
+## La flessibilità del software
+
+Intuitivamente parlando, sembra ovvio che noi sviluppatori, con una connessione internet e un portatile alla mano, possiamo essere enormemente più produttivi di decine di operai da catena di montaggio del secolo scorso. Ma perché è davvero così?
+
+Sicuramente è grazie al fatto che possiamo lavorare con strumenti frutto di decenni di evoluzione tecnologia esponenziale, ma la vera ragione, il vero discriminante, sta nel fatto nel "soft" in "software": il prodotto del nostro lavoro è spesso intangibile e facilmente manipolabile, tanto che se commettiamo un errore, non dobbiamo prendere il semilavorato e buttarlo via, sprecando materie prime, ma nella peggiore delle ipotesi abbiamo solo sprecato un po' di tempo.
+
+Tutto questo non è nient'altro che la "Terza rivoluzione industriale", come viene chiamata nei libri di scuola; dopo alcuni decenni in cui il collo di bottiglia era la potenza di calcolo, al giorno d'oggi abbiamo a disposizione CPU e memoria sufficienti per qualunque task mondano, e questa pura forza bruta si schianta ora invece contro problemi organizzativi: far lavorare decine, se non centinaia di sviluppatori assieme sullo stesso progetto software è un problema non banale, e che tanti prima di noi si sono trovati a dover risolvere.
+
+Uno strumento che è emerso vincitore da queste difficoltà è il "ciclo del feedback"; ma di che si tratta? E' un approccio in cui, in seguito di una semplice azione, si va a misurarne rapidamente l'effetto (il "feedback"), così da ottenerne nuove informazioni, che saranno impiegate per intraprendere i prossimi passi in maniera più conscia e informata, creando così il "ciclo". Questo ciclo generalmente permette di procedere in una attività a piccoli passi ma con una discreta rapidità e, cosa ancora più importante, sempre nella giusta direzione, visto che eventuali errori di rotta possono essere rapidamente corretti ad ogni passo.
+
+## Il manifesto Agile e l'Extreme Programming
+
+Alcuni movimenti che sono nati in seno alle difficoltà di questa rivoluzione sono il Manifesto Agile, l'Extreme Programming, il DevOps: il primo ad alto livello, il secondo ad un livello più "basso" tecnico, il terzo in maniera "mista", hanno tutti cercato di trovare modi per scrivere software in maniera più snella e organizzata, in modo che non ci si ritrovasse più in una melassa di linee di codice che, dopo pochi mesi di lavoro, diventava ingestibile.
+
+Alle fondamenta di questi approcci si trova **il ciclo del feedback**: vista la malleabilità del software e la relativa semplicità con cui si può evolvere e sperimentare, avere un ciclo del feedback breve ed efficace si è rapidamente dimostrato lo strumento vincente per uscire da quell'impasse.
+
+## Il TDD
```

Comment:
Ridurrei lo scope di questa sezione, indicando al letture che nell'apposito capitolo **Testing** si troveranno più informazioni riguardo a questa metodologia

Reply:
Ok, ma pensi abbia senso tenere la descrizione del ciclo? Quello mi sembra un collegamento necessario per spiegare perché lo cito..

Reply:
Absolutely, però non andrei in spiegazioni dettagliate perché, appunto, il capitolo sul testing ne parla ampiamente e ha senso rimandare il lettore lì piuttosto che raddoppiare un contenuto, però certo, senza dubbio è più che sensato metterlo!